resources/js/components/Product/AddToCart.vue

```diff
@@ -262,6 +258,25 @@ export default {
     },
 
     computed: {
+        currentThumbnail: function () {
+            return this.simpleProduct?.thumbnail || this.simpleProduct?.images?.[0] || this.product?.thumbnail
+        },
```

Comment:
This matches how this is handled on the PDP? Possible/useful to merge/extract so it's in one place?

Reply:
No, the PDP actually uses the `Images.vue` component where the `product-super-attribute-change` event. It's different also because on the PDP you have the whole gallery where here you only have the singular image.